inbm/cloudadapter-agent/fpm-template/usr/share/cloudadapter-agent/thingsboard/intel_manageability_widgets_version_3.3.json

```diff
@@ -1,154 +1,154 @@
+        "settingsSchema": "{}",
+        "dataKeySettingsSchema": "{}\n",
+        "defaultConfig": "{\"datasources\":[{\"type\":\"function\",\"name\":\"My Device\",\"dataKeys\":[{\"name\":\"f(x)\",\"type\":\"function\",\"label\":\"Happiness\",\"color\":\"#ff9800\",\"settings\":{},\"_hash\":0.26790778871949805,\"funcBody\":\"return Math.round(1000*Math.cos(time/5000));\"},{\"name\":\"f(x)\",\"type\":\"function\",\"label\":\"Temperature\",\"color\":\"rgb(33, 150, 243)\",\"settings\":{},\"_hash\":0.682878536509057,\"funcBody\":\"var value = prevValue + Math.random() * 100 - 50;\\nvar multiplier = Math.pow(10, 2 || 0);\\nvar value = Math.round(value * multiplier) / multiplier;\\nif (value < -1000) {\\n\\tvalue = -1000;\\n} else if (value > 1000) {\\n\\tvalue = 1000;\\n}\\nreturn value;\"}]}],\"timewindow\":{\"realtime\":{\"timewindowMs\":60000}},\"showTitle\":false,\"backgroundColor\":\"#fff\",\"color\":\"rgba(0, 0, 0, 0.87)\",\"padding\":\"0\",\"settings\":{},\"title\":\"Device List\",\"dropShadow\":true,\"enableFullscreen\":false,\"widgetStyle\":{},\"titleStyle\":{\"fontSize\":\"16px\",\"fontWeight\":400},\"useDashboardTimewindow\":true,\"displayTimewindow\":true,\"showLegend\":false,\"actions\":{}}"
+      }
+    },
+    {
+      "alias": "gavin_ota_form1_show_hide",

```

Comment:
can we remove my name from this?

Reply:
I think the PR that Harsha still has open fixes the name issue.  Maybe that should be tested and merged first so that get removed in this one too.

Reply:
This is the PR:  https://github.com/intel/intel-inb-manageability/pull/28.  I think we just never merged it because we weren't sure if it was tested yet.